LUIS/Authoring SDK/LUIS-Authoring.Sample/Pages/BookingApp/StartPage.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Microsoft.Azure.CognitiveServices.LUIS.Authoring.Sample.Pages.BookingApp
```

Comment:
I recommend to change folder names for consistent naming conversion.
Authoring SDK  ==> Authoring
LUIS-Authoring.Sample ==> LUIS.Authoring.Sample